server/models/procedures/stock.sql

```diff
@@ -375,7 +375,8 @@ CREATE PROCEDURE `computeStockQuantity` (
   SET _start_date = DATE(_start_date);
   SET @depot_number = 0;
 
-  DELETE FROM `temp_depot`;
+  DROP TEMPORARY TABLE IF EXISTS `stock_movement_status`;
```

Comment:
Do you need this in the live code, or with this a one-time fix for your machine?

If it is a one-time fix, I would prefer it be put in the `migrate.sql` file.

Reply:
I had an existing temporary table named ```stock_movement_status```.
this instraction is no important in production. I should remove this line. Thanks